### DISCUSSION TITLE
Open Telemetry

### DISCUSSION THREAD

**Discussion by muneernet** [open-telemetry/community#735] [Community Discussions]
I  have a  .net core application hosted on  http://localhost:789/Weatherservice now i want to  configure open telemetric collector agent to this . Could you let me know what to configure in yaml file attaching in the thread . do i need to configure in reciver section  

receivers:
  jaeger:
    protocols:
      thrift_http:
        endpoint: "0.0.0.0:14268"
  otlp:
   protocols:
      grpc: # on port 55680
      http: http://localhost:789/Weatherservice
[collector yam;.txt](https://github.com/open-telemetry/community/files/6465942/collector.yam.txt)
 Any help highly appreciable

**Comment by SergeyKanzhelev** [comment]
Use this exporter: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol

And configure `otlp` grpc receiver: https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver

CC: @open-telemetry/dotnet-approvers

**Reply by rkbodenner** [reply]
Hi @muneernet, wondering if this answered your question and whether you could mark it as Answered. If not, how else could we help? Thanks!
